clicache/src/impl/CacheWriter.hpp

```diff
@@ -76,8 +76,7 @@ namespace Apache
             RegionEvent<TKey, TValue> gevent(ev->GetNative());
             return m_writer->BeforeRegionDestroy(%gevent);
           }
-          
-          virtual void Close(Apache::Geode::Client::Region<Object^, Object^>^ region) override
+          virtual void Close(Apache::Geode::Client::Region<Object^, Object^>^ region)
```

Comment:
Why did the override get dropped here?

Reply:
it was very wrong in terms of matching the actual prototype, and I was removing a warning. I actually didn't intend to check this in.

Reply:
I know it is supposed to do that if someone in advertently changes the base function prototype, that is why I didn't intend to check that in.